yarn-project/aztec/docker-compose.yml

```diff
@@ -12,7 +12,7 @@ services:
         exec anvil -p 8545 --host 0.0.0.0 --chain-id 31337 --silent
       fi'
     ports:
-      - '${SANDBOX_ANVIL_PORT:-8545}:8545'
```

Comment:
Why also removing this? 👀

Reply:
I don't think this docker compose is used anywhere, but I'll just provide the same options as the sandbox one (the one provided in `aztec-up`)